### ISSUE TITLE
Method links in docs don't work on Windows

### ISSUE THREAD

**Issue by yha** [julia-vscode/julia-vscode#1944]
e.g., clicking on the method link in the hover docs for `ProgressLogging.@progress`:
![image](https://user-images.githubusercontent.com/4170948/107978268-254d5e80-6fc5-11eb-8596-c3cb9c0bc66c.png)

Clicking the same link from the docs pane fails silently.
The link display in both cases omits the backslash in front of ".julia":
![image](https://user-images.githubusercontent.com/4170948/107978556-a7d61e00-6fc5-11eb-9e62-a95e4b6c39cc.png)

**Comment by jebej** [comment]
I can reproduce both issues.

**Comment by pfitzseb** [comment]
Yeah same. Turns out we never tested this particularly well on Windows :)

**Comment by davidanthoff** [comment]
The language server backend was rewritten in v1.220.0: StaticLint.jl and SymbolServer.jl were folded into JuliaWorkspaces, and completion, hover, signature help, go-to-definition and find-references were reimplemented. This report predates that rewrite, so the code that produced it no longer exists and the behaviour cannot be confirmed against the current engine.

Closing as outdated. If you still see this on a current release, please open a fresh issue with a reproducer and we will look at it against the new engine.
